### PR TITLE
Brig Physician now requires medical playtime

### PR DESCRIPTION
**Pull request by maxspells**
## About The Pull Request

Changes the required playtime for brig physician from 30 hours of security to 12 hours of medical. If there is a way to require time played in both departments, I'd much rather do that but this is my first PR and I'm mostly just trying to learn how PRs work. Feel free to deny!

## Why It's Good For The Game

Whenever I hear complaints about brig docs it's either:
A. They suck at doing medical work
B. They gear up and try to act like a regular sec instead of focusing on medical work
C. They stand around doing nothing because (I assume) they don't know what they are doing.

Forcing them to have a little bit of MD background will hopefully ensure they go into the job with at least some basic medical knowledge, and hopefully stave off people picking Brig Physician whenever there are no sec slots open and they want to just play sec.

### CODE REVIEW

File: fulp_modules/main_features/jobs/brigdoc/code/brigdoc.dm [hunk 1]
```diff
@@ -8,10 +8,12 @@
...
 	minimal_player_age = 14
-	exp_requirements = 300
-	exp_granted_type = EXP_TYPE_SECURITY
+	exp_requirements = 120
+	exp_required_type = EXP_TYPE_CREW
+	exp_required_type_department = EXP_TYPE_MEDICAL
+	
 	fulp_spawn = /obj/effect/landmark/start/brigdoc
-
+
```
Comment (SgtHunk) [inline fulp_modules/main_features/jobs/brigdoc/code/brigdoc.dm:16]: what

Comment (Dr-Pope) [reply fulp_modules/main_features/jobs/brigdoc/code/brigdoc.dm:16]: Quality addition

Comment (maxspells) [reply fulp_modules/main_features/jobs/brigdoc/code/brigdoc.dm:16]: I'm bad and it feels bad

**Comment by Nobelium-XVIII** [comment]
Merge conflict....

File: fulp_modules/main_features/jobs/brigdoc/code/brigdoc.dm (hunk 1 above)
Comment (maxspells) [reply fulp_modules/main_features/jobs/brigdoc/code/brigdoc.dm]: should be good now

**Review by SgtHunk — APPROVED** [review]
Eh, seems good enough
Brig doc should be removed and replaced with a different job but the issue is figuring out what job does the station need that isn't fulfilled by anyone else.

**Pull request merged**